README.md

```diff
@@ -1,2 +1,57 @@
-# portfolio
-This is my portfolio project
+# hello-microverse-project-
+
+> This project was made using flexbox and it was based on Template 3. The aim of the project is to build a personal portfolio site
+
+
+## Built With
+
+- HTML
+- CSS
+- linter
+
+## Live Demo (if available)
+
+No
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Flexbox
+To run this project, install it locally using npm:
+
+
+### Install
+
+$ cd ../lorem
+$ npm install
+$ npm start
+
+
+## Authors
+
+👤 **David Tamayo**
+
+- GitHub: [@IngDavidTM](https://github.com/IngDavidTM)
+- Twitter: [@David5TM](https://twitter.com/David5TM)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ing-david-tamayo)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check
+
```

Comment:
- [ ] Please add a link for contribution in case someone would like to contribute to your project or want to reveal a bug:smile: